src/main/java/com/savvato/tribeapp/services/AttributesServiceImpl.java

```diff
@@ -75,4 +85,26 @@ public boolean isGivenAdverbFound(String adverb) {
     public boolean isGivenPrepositionFound(String preposition) {
         return this.prepositionRepository.findByWord(preposition).isPresent();
     }
+
+    @Override
+    public Optional<List<AttributeDTO>> getAttributesByUserId(Long userId) {
+
+        List<AttributeDTO> attributes = new ArrayList<>();
+
+        // Get all user phrases as phraseDTOs
+        Optional<List<PhraseDTO>> optUserPhraseDTOs = phraseService.getListOfPhraseDTOByUserId(userId);
+
+        // If there are phrases, build DTO and add to attributes list
+        if(optUserPhraseDTOs.isPresent()) {
+            List<PhraseDTO> attribute = optUserPhraseDTOs.get();
```

Comment:
Probably a better variable name here is `list`, instead of `attribute`.

Reply:
In the code I submitted, this list name of "attribute" should satisfy the "attribute" in the print out. I assume we may want to revisit the naming of attributes, attribute, etc. if we incorporate more attributes other than phrases. For now, do you want me to keep that print out of "attribute" or change the requirement? Or another option I'm not considering?

```
{
  "attributes": {
    "attribute": {
      "id" : 2323,
      "phrase": {
        "adverb": ...
        "verb": ...
        "preposition": ...
        "noun": ...
      }
    },
    "attribute": {
      "id" : 7822,
      "phrase": {
        "adverb": ...
        "verb": ...
        "preposition": ...
        "noun": ...
      }
    }
  }
}
```